ibtk/examples/VCViscousSolver/input2d

```diff
@@ -0,0 +1,163 @@
+u {
+   function_0 = "X_0^2*X_1^2"
+   function_1 = "X_0*X_1"
+}
+
+f {
+   function_0  = "1.0 + 2.0*X_0*X_1 + 2*X_0^2 + 4.0*X_1^2 + 4.0*X_0^3*X_1 + 8.0*X_0*X_1^3"
+   function_1  = "2.0*X_0^2 + X_1^2 + 4.0*X_0*X_1 + 6.0*X_0^2*X_1^2"
+}
+
+mu {
+   function = "1.0 + X_0*X_1"
+}
+
+// u velocity
+VelocityBcCoefs_0 {
+
+   u_fcn = "X_0^2*X_1^2"
+      
+   acoef_function_0 = "1.0"
+   acoef_function_1 = "1.0"
+   acoef_function_2 = "1.0"
+   acoef_function_3 = "1.0"
+
+   bcoef_function_0 = "0.0"
+   bcoef_function_1 = "0.0"
+   bcoef_function_2 = "0.0"
+   bcoef_function_3 = "0.0"
+
+   gcoef_function_0 = u_fcn
+   gcoef_function_1 = u_fcn
+   gcoef_function_2 = u_fcn
+   gcoef_function_3 = u_fcn
+   
+}
+
+// v velocity
+VelocityBcCoefs_1 {
+
+   v_fcn = "X_0*X_1"
+   
+   acoef_function_0 = "1.0"
+   acoef_function_1 = "1.0"
+   acoef_function_2 = "1.0"
+   acoef_function_3 = "1.0"
+
+   bcoef_function_0 = "0.0"
+   bcoef_function_1 = "0.0"
+   bcoef_function_2 = "0.0"
+   bcoef_function_3 = "0.0"
+
+   gcoef_function_0 = v_fcn
+   gcoef_function_1 = v_fcn
+   gcoef_function_2 = v_fcn
+   gcoef_function_3 = v_fcn
+
+}
+
+
+
+solver_type = "VC_VELOCITY_PETSC_KRYLOV_SOLVER"
```

Comment:
Why is the basic PETSC_KRYLOV_SOLVER not OK here? Does there really need to be a specialization of this particular class?

Reply:
Basically the SCPoissonManager creates a vector-valued Laplace operator by default. This way, I defining a function in driver (eventually in integrator) to create a PETScKrylovSolver with correct operator. I thought the original design of adding solvermaker() was for this purpose. Do you have some other suggestion?